docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -7,20 +7,125 @@ slug: /project/rewards-troubleshooting
 
 A guide to interpreting mobile app error messages, understanding their meanings, and taking appropriate actions to resolve them.
 
-### No wallet
+## No wallet
 
+It seems that you haven't connected a wallet to your account yet. Please note that in order to receive rewards, you must connect a wallet address. Here's what you need to do:
 
-### No Station Data 
+1. First, learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
+2. Then, follow the instructions for [joining Arbitrum Sepolia and see your $WXM tokens](/wallet/how-to-join-arbitrum-and-see-your-tokens#add-arbitrum-sepolia-network-to-metamask).
 
-### Sensor Problems
+Once you've completed the above steps, you'll be all set to claim your rewards. Check back tomorrow and see "[How to Claim Rewards](/project/rewards-claiming)" for further guidance.
 
-### Weather Data Gaps
+<hr/>
 
-### Bad Station Deployment
+## No Station Data
 
-### No location Data
+It seems that your station sent no data at all. 
+That either means that your station was offline or not connected to the network. 
 
-### Location Not Verified
+Here's what you need to do:
 
+### For the WS-1000 (WiFi/M5 version)
 
-### User Relocation Penalty
\ No newline at end of file
+1. Check if the station's battery level is low through mobile app settings.
```

Comment:
As I suggested in the Linear task, before starting that list, we need to include a screenshot like this one [here](https://docs.weatherxm.com/img/wxm-devices/wifi-m5-bundle/m5-ui-new.png), showing the M5 and explaining what each icon at the top right corner means.

Reply:
We definitely need this to be added, if not now, at least until the end of the week. I have seen numerous messages on Discord from users not understanding what each icon means and we have nowhere that piece of information in our docs.